tools/mksurfdata_map/src/mkpctPftTypeMod.F90

```diff
@@ -511,6 +510,49 @@ end subroutine check_vals
   ! Module-level routines (not member functions)
   ! ========================================================================
 
+  !-----------------------------------------------------------------------
+  subroutine set_max_p2l_array(pct_pft_max_arr,pct_pft_arr)
```

Comment:
I'd rename "set" to "update" in this subroutine name, to indicate that it is (possibly) updating existing max values.